src/test/java/com/project/playlist/service/impl/GetSortedVideosForPlaylistTest.java

```diff
@@ -0,0 +1,115 @@
+package com.project.playlist.service.impl;
+
+import com.project.playlist.exceptions.PlaylistNotFoundException;
+import com.project.playlist.model.Playlist;
+import com.project.playlist.model.PlaylistVideo;
+import com.project.playlist.model.Video;
+import com.project.playlist.repository.PlaylistVideoRepository;
+import com.project.playlist.service.PlaylistService;
+import com.project.playlist.service.PlaylistVideoService;
+import com.project.playlist.service.VideoService;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.api.function.Executable;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.*;
+
+@SpringBootTest
+@ExtendWith(MockitoExtension.class)
+public class GetSortedVideosForPlaylistTest {
```

Comment:
Rename class to `PlaylistVideoServiceTest` since you will test whole unit in this class